internal/ethapi/override/override_test.go

```diff
@@ -37,7 +36,7 @@ func (p *precompileContract) Run(input []byte) ([]byte, error) { return nil, nil
 
 func TestStateOverrideMovePrecompile(t *testing.T) {
 	db := state.NewDatabase(triedb.NewDatabase(rawdb.NewMemoryDatabase(), nil), nil)
-	statedb, err := state.New(types.EmptyRootHash, db)
+	statedb, err := state.New(common.Hash{}, db)
```

Comment:
Looks like this change should be its standalone PR?

Reply:
Yeah I don't know why that was there :)